website/cue/reference/components/transforms/remap.cue

```diff
@@ -89,7 +96,9 @@ components: transforms: "remap": {
 			common:   false
 			required: false
 			description: """
-				Drop the event if the VRL program returns an error at runtime.
+				Drop the event from the primary output stream if the VRL program returns
+				an error at runtime. These events will instead be written to the
+				`errors` output.
```

Comment:
Same here: re: runtime error or explicit abort, and stream is named `dropped` now.